src/components/InvoicesList/InvoicesList.js

```diff
@@ -56,21 +74,7 @@ const InvoicesList = ({ dependencies: { dopplerBillingApiClient } }) => {
       </Helmet>
       <HeaderSection>
         <div className="col-sm-12 col-md-12 col-lg-12">
-          <nav className="dp-breadcrumb">
-            <ul>
-              <li>
-                <a href={_('invoices_list.control_panel_account_preferences_url')}>
-                  {_('invoices_list.control_panel_section')}
-                </a>
-              </li>
-              <li>
-                <a href={_('invoices_list.control_panel_billing_information_url')}>
-                  {_('invoices_list.control_panel_billing_information_section')}
-                </a>
-              </li>
-              <li>{_('invoices_list.title')}</li>
-            </ul>
-          </nav>
+          <Breadcrumb />
```

Comment:
Que opinás de hacer algo así para que luego se puedan reusar los componentes en otras páginas?

```suggestion
          <Breadcrumb>
            <BreadcrumbItem href={_('invoices_list.control_panel_account_preferences_url')}>
              {_('invoices_list.control_panel_section')}
            </BreadcrumbItem>
            <BreadcrumbItem href={_('invoices_list.control_panel_billing_information_url')}>
              {_('invoices_list.control_panel_billing_information_section')}
            </BreadcrumbItem>
            <BreadcrumbItem current>
              {_('invoices_list.title')}
            </BreadcrumbItem>
          </Breadcrumb>
```

Reply:
Let me check it

Reply:
I implemented a first approach about it. What do you think?